packages/amplify-app/package.json

```diff
@@ -32,6 +32,7 @@
     "@aws-amplify/amplify-frontend-flutter": "1.4.0",
     "@aws-amplify/amplify-frontend-ios": "3.6.1",
     "@aws-amplify/amplify-frontend-javascript": "3.8.1",
+    "@types/glob": "^7.1.1",
```

Comment:
Can this be a devDependency?

Reply:
it's main codebase
https://github.com/aws-amplify/amplify-cli/blob/e5f00c392b150bbb47aed30c81cb5982b17c51a2/packages/amplify-app/src/xcodeHelpers.js#L2

Reply:
That's `glob` itself. The types are only consumed at build time, no?